package.json

```diff
@@ -3,46 +3,28 @@
   "version": "1.0.0",
```

Comment:
I'd change your version here to '0.0.1', or something else that indicates that this is very much in a "pre-release" state.